Dockerfile.build

```
@@ -1,13 +1,13 @@
 FROM golang:1.15-alpine AS builder
-ENV GO111MODULE=on
+ENV GO111MODULE on

```

why have you changed it to the alterative syntax while the next line is still using the default syntax with the `=` ?
```suggestion
ENV GO111MODULE=on
```

---

It's the official syntax

---

official syntax is defined [here](https://docs.docker.com/engine/reference/builder/#env) 
Anyway we have 2 ENV definitions in this file with 2 different syntaxes ;-)
it would make sense to keep is consistent